dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/DefaultArguments.java

```diff
@@ -77,6 +82,46 @@ public String valuePattern() {
     }
   }
 
+  public static class DurationValueConverter implements ValueConverter<Duration> {
+
+    private enum AllowedUnits {
+      HOUR(HOURS, "hourly"),
+      DAY(DAYS, "daily");
+
+      private final ChronoUnit chronoUnit;
+      private final String commandLineFlag;
+
+      AllowedUnits(ChronoUnit chronoUnit, String commandLineFlag) {
+        this.chronoUnit = chronoUnit;
+        this.commandLineFlag = commandLineFlag;
+      }
+    }
+
+    public static DurationValueConverter INSTANCE = new DurationValueConverter();
+
+    private DurationValueConverter() {}
+
+    @Override
+    public Duration convert(String value) {
+      for (AllowedUnits unit : AllowedUnits.values())
+        if (unit.commandLineFlag.equals(value)) return unit.chronoUnit.getDuration();
+      throw new ValueConversionException("Not a valid unit of interval: " + value);
+    }
+
+    @Override
+    public Class<? extends Duration> valueType() {
+      return Duration.class;
+    }
+
+    @Override
+    public String valuePattern() {
+      return Arrays.stream(AllowedUnits.values())
+          .map(unit -> unit.commandLineFlag)
+          .map(String::toLowerCase)
+          .collect(Collectors.joining("/"));
```

Comment:
Optional: I would suggest to join the values by commas (`", "`) instead of slashes for better UX. Also in `BooleanValueConverter`.